blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -25,13 +27,24 @@ const EmptyState = ({ repoName }) => (
                 Commit to the repository <em>{repoName}</em> or run the pipeline manually.
             </p>
 
-            <button>Run Now</button>
+            {showRunButton && <RunNonMultiBranchPipeline pipeline={pipeline} buttonText="Run Now" />}
         </EmptyStateView>
     </main>
 );
 
 EmptyState.propTypes = {
     repoName: string,
+    pipeline: object,
+    showRunButton: boolean,
+};
+
+const RunNonMultiBranchPipeline = ({ pipeline, buttonText }) => (
+    <RunPipeline organization={pipeline.organization} pipeline={pipeline.name} buttonClass="btn-primary inverse non-multi-branch" buttonText={buttonText} />
```

Comment:
This `btn-primary inverse` style will look a little funny on the current empty state (which is green), but will look fine once JDL `0.0.59` is released, as it changes the empty state to be the primary blue colour (nice !!).